### ISSUE TITLE
HDBits categories (easy fix IMHO)

### ISSUE THREAD

**Issue by LSL1337** [Radarr/Radarr#1419]
Audio Track Only torrents are part of the normal search results from HDB
these torrents only contain the audio track, which can be remuxed to other releases (useful for VERY small number of users, not really relevant for automated movie downloading)

For example:
If you search for 'Patriots Day' in radarr, and have HDB enabled, it will give you the following result (among many other files):

Patriots Day 2016 Blu-ray English DTS-HD MA 7.1	[4.0GB]

radarr tagged it as bluray-720p. It has blu ray in the name, but no resolution. maybe cos of the size, it was ranked as 720p, or is it the baseline bluray quality, i don't know, not really important.

In the api, there is number for category.
the above mentioned release is category:6
I don't know how to access the API, but on the normal HDB web interface, they have the following categories:
1 - Movie
2 - TV
3 - Docu
4 - Music video (could be concert rips as well (from blu ray)
5 - Sports
6 - Audio Track
7 - XXX
8 - Misc (Extras, bonus disk, uhd demo, premier fotage (red carpet), etc)
9,10 - nothing, so i guess there is only 8.

i think category 2, 5,6,7 should be disabled, maybe 8 as well.
i'd leave 1, 3,4 and 8 in. I didn't yet noticed hits from these categorys, but in same cases they might be usefull.
well, a full movie will never be in cat 8, and if i want to grab extras, i won't use radarr for that anyway, so maybe useless to leave 8 in as well.

TL;DR:
HDB search results from category 2,5,6,7 should be disabled, but 6 is the only one that counts.

cheers

**Comment by randellhodges** [comment]
I was thinking of trying my hand at making this modification. Would there be value to make this configurable? 

While making this change, would there be value to adding other filtering condition?

category
codec
medium
1 Blu-ray/HD DVD , 3 Encode, 4 Capture, 5 Remux, 6 WEB-DL

I am thinking of implementing medium because I don't do full bluray downloads, only encodes of blurays. Right now, it will match against the full bluray and not an encode.

**Comment by LSL1337** [comment]
it would be preferable to put releases into categories based on HDB Meta data (categoria, type, codec), as the current mode, which is based on regex.
So this would be a farily big modification, if you want to support it for profiles.
but i guess a simple checkbox on these dimensions would be better than a hardcoded one, which i descried in the OP.
choice is always better imho

**Comment by randellhodges** [comment]
I agree for categories, the real long term solution would be to tie it to profiles. That is probably more than I'd want tackle at the moment.

I wonder if I just do it on a filter approach, what the chance are of getting it merged?

Also, unless I want to create new UI controls, I am somewhat limited to how to select these values. I think each item would need to be a On/Off slider (I believe that is how they render a checkbox).

If we did categories, that would be 8 of them. If I put in Codec, that would be 4 more, and medium that would be another 5 for a total UI bloat of 17 (and storing 17 more bools in the settings!)

Would be something like a checklist, but I'm not really up for creating new UI components right now.

If we didn't want a bunch of checkboxes, another, almost equally unfriendly way, would be to put in a textbox and let the user put in a comma delimited list of integers.

I'll take a quick peek at the UI piece and see if I can come up with something that doesn't go to far down a rabbit hole.

**Comment by galli-leo** [comment]
This is fixed on latest release.
